install_config/prometheus_cluster_monitoring.adoc

```diff
@@ -0,0 +1,15 @@
+:toc: macro
+:toc-title:
+
+toc::[]
+
+[[cluster-monitoring]]
+= Prometheus cluster monitoring
+
+include::install_config/monitoring/overview.adoc[leveloffset=+1]
+include::install_config/monitoring/configuring-openshift-cluster-monitoring.adoc[leveloffset=+1]
+include::install_config/monitoring/configuring-alertmanager.adoc[leveloffset=+1]
+include::install_config/monitoring/alerting-rules.adoc[leveloffset=+2]
+include::install_config/monitoring/configuring-etcd-monitoring.adoc[leveloffset=+1]
+include::install_config/monitoring/accessing-prometheus-alertmanager-grafana.adoc[leveloffset=+1]
+include::install_config/monitoring/update-and-compatibility-guarantees.adoc[leveloffset=+1]
```

Comment:
I think this information should go after the overview or in the overview itself.